content/css/jenkins.css

```diff
@@ -26,7 +60,8 @@ div .authors {display:block; padding:0; margin-bottom:3rem}
 .center > .center {display:inline; float:none; display:inline-block}
 body .jumbotron {margin:0}
 body .card-block {display:block}
-.jumbotron.docker {box-shadow:inset 0 0 150px rgba(0,0,0,.3)}
+.jumbotron.docker {background:#042E54 url(/images/whale_1030x500.jpg) fixed no-repeat right}
```

Comment:
Are we eligible to use a non-classic Docker image here? Could it cause trademark issues?